package/yast2-auth-client.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
```

Comment:
Hmmm, such a huge change in the code and in the version number and the only modification in the changelog file is the order of an entry. I'd expect some more information here :-)